packages/block-editor/src/components/block-navigation/list-item.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, VisuallyHidden } from '@wordpress/components';
+import {
+	__experimentalGetBlockLabel as getBlockLabel,
+	getBlockType,
+} from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../block-icon';
+
+export default function BlockNavigationListItem( {
+	block,
+	onClick,
+	isSelected,
+	wrapperComponent: WrapperComponent,
+	children,
+} ) {
+	const blockType = getBlockType( block.name );
+
+	return (
+		<div className="block-editor-block-navigation__item">
```

Comment:
I just noticed some of the class names don't seem to be right after moving the code around. The root element should have the class name `block-editor-block-navigation-list-item` to reflect the component name.